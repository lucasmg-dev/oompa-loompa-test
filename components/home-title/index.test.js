import '@testing-library/jest-dom'

import { HomeTitle } from './index'
import { render } from '@testing-library/react'

it('renders correctly', () => {
  const { asFragment } = render(<HomeTitle />)
  expect(asFragment()).toMatchSnapshot()
})
