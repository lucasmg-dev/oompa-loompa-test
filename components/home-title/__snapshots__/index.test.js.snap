// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <div
    class="homeTitle"
  >
    <h1
      class="h1"
    >
      Find your Oompa Loompa
    </h1>
    <h2
      class="h2"
    >
      There are more than 100k
    </h2>
  </div>
</DocumentFragment>
`;
